app/models/game.rb

```diff
@@ -30,4 +30,37 @@ def populate_game!
   def piece_at(x,y)
     pieces.find_by(x_coord: x, y_coord: y) || EmptySquare.new
   end
+
+  def in_check? 
+    if white_in_check == true || black_in_check == true
```

Comment:
I really like how you break it up into 2 separate methods! Could we make it a bit more concise and have the method be one line with `white_in_check || black_in_check`? 😄

Reply:
Good idea! I'll make the commit now.